scripts/parse_gestalt.py

```diff
@@ -7,37 +7,52 @@
 specific TAD pathway genes
 
 Usage:
-Command line 'python scripts/parse_gestalt.py -t <TRAIT>'
+Command line 'python scripts/parse_gestalt.py' with the required flag:
 
-    -t <TRAIT> is predefined by the manual WebGestalt step (see README)
-    e.g. <TRAIT> is BMD for Bone Mineral Density
+    --trait              is predefined by the manual WebGestalt step
+                         (see README). Or it can be specified by the name of
+                         the gestalt output file name.
+                         Example 1: "BMD" for Bone Mineral Density GWAS
+                         Example 2: "custom_group" for file named -
+                                    "data/gestalt/custom_group_gestalt.tsv"
+
+And the optional flag:
+
+    --process_type      Defaults to update, if specified then
 
 Output:
-pickle files with pathway specific dictionaries and summary of pathway p values
+csv files with pathway specific information and summary of pathway p values
 """
 
 import os
 import argparse
 import pandas as pd
 import io
+import re
 
 
-def read_gestalt(fh):
+def read_gestalt(file_name, process):
     """
     Read in the gestalt data and parse pathways
 
     Arguments:
-    :param fh: the location of the file to read
+    :param file_name: the location of the file to read
+    :param process: the type of Gestalt file to parse
 
     Output:
     1) Pandas Dataframe of entire TRAIT pathways
     2) Pathway summary text file
     """
 
     # Read in the entire file as string
-    with open(fh, 'r') as trait_fh:
+    with open(file_name, 'r') as trait_fh:
         gestalt = trait_fh.read()
 
+    if process:
```

Comment:
It is a result of a very sloppy custom file that outputs as a result of a [Webgestalt analysis](http://www.webgestalt.org/).